diaphora_ida.py

```diff
@@ -2806,6 +2845,51 @@ def main():
       os.remove("%s-crash" % file_out)
 
     idaapi.qexit(0)
+  # EXPORT - works with pseudocode, better than above
+  elif os.getenv("DIAPHORA_AUTO2") is not None:
+    if g_debug:
+      log("Handling DIAPHORA_AUTO2")
+      log("DIAPHORA_EXPORT_FILE=%s" % os.getenv("DIAPHORA_EXPORT_FILE"))
+    file_out = os.getenv("DIAPHORA_EXPORT_FILE")
+    if file_out is None:
+      raise Exception("No export file specified!")
+    _diff_or_export(False, file_out=file_out)
+    idaapi.qexit(0)
+  # DIFF-SHOW
+  elif os.getenv("DIAPHORA_AUTO4") is not None:
+    if g_debug:
+      log("Handling DIAPHORA_AUTO4")
+      log("DIAPHORA_AUTO4=%s" % os.getenv("DIAPHORA_AUTO4"))
+      log("DIAPHORA_DB1=%s" % os.getenv("DIAPHORA_DB1"))
+      log("DIAPHORA_DB2=%s" % os.getenv("DIAPHORA_DB2"))
+      log("DIAPHORA_DIFF=%s" % os.getenv("DIAPHORA_DIFF"))
+      log("DIAPHORA_EA1=%s" % os.getenv("DIAPHORA_EA1"))
+      log("DIAPHORA_EA2=%s" % os.getenv("DIAPHORA_EA2"))
+      log("DIAPHORA_HTML_ASM=%s" % os.getenv("DIAPHORA_HTML_ASM"))
+      log("DIAPHORA_HTML_PSEUDO=%s" % os.getenv("DIAPHORA_HTML_PSEUDO"))
+    db1 = os.getenv("DIAPHORA_DB1")
+    if db1 is None:
+      raise Exception("No database file specified!")
+    db2 = os.getenv("DIAPHORA_DB2")
```

Comment:
It seems it's never used?

Reply:
Correct, it is not used atm and could be removed. I wasn't sure if we would not need it at some point hence why I left it, but there is a comment about that actually. Feel free to remove it if you don't want it there, though.